lib/src/DateTimePicker/DateTimePicker.jsx

```diff
@@ -144,7 +151,7 @@ export class DateTimePicker extends Component {
             />
         }
 
-        <div className={classes.viewContainer}>
+        <ViewContainerComponent className={classes.viewContainer} {...ViewContainerComponentProps}>
```

Comment:
why its only wrap the year?

Reply:
It wraps exactly the same things as before. GitHub collapses the diff, so that it looks like it only contains one thing.